packages/web3/test/integration/main_web3.test.ts

```diff
@@ -0,0 +1,249 @@
+/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+/* eslint-disable jest/no-standalone-expect */
+import Contract from 'web3-eth-contract';
+import { ENS } from 'web3-eth-ens';
+import Web3Eth from 'web3-eth';
+import { JsonRpcOptionalRequest, Web3BaseProvider } from 'web3-common';
+import HDWalletProvider from '@truffle/hdwallet-provider';
+import { Web3 } from '../../src/index';
+
+import { erc20Abi } from '../fixtures/erc20';
+
+import {
+	getSystemTestProvider,
+	describeIf,
+	getSystemTestAccounts,
+	accounts as staticAccounts,
+} from '../fixtures/system_test_utils';
+import { externalHttpsStringProvider, externalWssStringProvider } from '../fixtures/config';
+
+describe('Web3 instance', () => {
+	let clientUrl: string;
+	let accounts: string[];
+	beforeAll(async () => {
+		clientUrl = getSystemTestProvider();
+		accounts = await getSystemTestAccounts();
+	});
+	describeIf(getSystemTestProvider().startsWith('http'))(
+		'Create Web3 class instance with http string providers',
+		() => {
+			it('should create instance with string provider', async () => {
+				const web3 = new Web3(clientUrl);
+				expect(web3).toBeInstanceOf(Web3);
+			});
+			it('should create instance with string of external http provider', async () => {
+				const web3 = new Web3(externalHttpsStringProvider);
+				expect(web3).toBeInstanceOf(Web3);
+			});
+			// todo fix ipc test
+			// https://ethereum.stackexchange.com/questions/52574/how-to-connect-to-ethereum-node-geth-via-ipc-from-outside-of-docker-container
```

Comment:
we can skip test for instantiation with IPC provider now in alpha release, create a new issue for that.